Patches/Metal Gear Rimworld-Gekko/ThingDefs_Races/Races_Mechanoid.xml

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Metal Gear Rimworld-Gekko</li>
+		</mods>
+
+		<match Class="PatchOperationSequence">
+			<operations>
+
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="MGR_Mech_Gekko"]/statBases</xpath>
+					<value>
+						<PartialArmorBody>0</PartialArmorBody>
+					</value>
+				</li>
+
+				<!--Consistency with vanilla mechs-->
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="MGR_Mech_Gekko"]/statBases</xpath>
+					<value>
+						<SmokeSensitivity>0</SmokeSensitivity>
+						<NightVisionEfficiency>0.80</NightVisionEfficiency>
+					</value>
+				</li>
+
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="MGR_Mech_Gekko"]/statBases</xpath>
+					<value>
+						<ArmorRating_Heat>0.25</ArmorRating_Heat>
+						<CarryWeight>250</CarryWeight>
```

Comment:
Because the Gekko has a bodySize of 4, it can carry 1000 kg (250 * 4). I'd go ahead and reduce this from value 250 to 60, so that it gets an appropriate carry weight with bodySize factored in.